packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -61,27 +61,26 @@ const useStyles = makeStyles({
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'start',
-    ...shorthands.gap(tokens.spacingVerticalS),
+    gap: tokens.spacingVerticalS,
   },
   icons: {
     display: 'grid',
     gridTemplateColumns: 'auto auto',
     gridTemplateRows: 'auto auto',
-    gridRowGap: tokens.spacingVerticalS,
-    gridColumnGap: tokens.spacingHorizontalS,
+    rowGap: tokens.spacingVerticalS,
```

Comment:
Probably a change for a separate PR but these rules can be collapsed to just `gap` ya?

Reply:
Cool. I'll make that change once this PR is merged.